src/App.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react';
 import { createBrowserRouter } from 'react-router-dom';
-import Home from './pages/Home/Home';
+// import Home from './pages/Home/Home';
```

Comment:
This is one of the comments. please go through the rest of the code to ensure it aligns with each requirement of the Merge review checklist